### PR TITLE
docs: add documentation around component media

### PR DESCRIPTION
**Pull request by JuroOravec**
Adds missing documentation for https://github.com/EmilStenstrom/django-components/pull/870

### CODE REVIEW

File: docs/css/style.css [hunk 1]
```diff
@@ -11,6 +11,19 @@ h6 {
   padding-top: 40px;
 }
 
+.md-typeset h3 {
```
Comment (JuroOravec) [inline docs/css/style.css:14]: Added this so that there's a horizontal line even between `<h3>` headers

File: src/django_components/__init__.py [hunk 2]
```diff
@@ -44,6 +45,8 @@
     "Component",
     "ComponentFileEntry",
     "ComponentFormatter",
+    "ComponentMediaInput",
+    "ComponentMediaInputPath",
```
Comment (JuroOravec) [inline src/django_components/__init__.py:49]: Exported these, so that the typings for `Media.js` and `Media.css` show up in the API reference.

**Comment by JuroOravec** [comment]
Some examples:

Screenshot 2024-12-30 at 10 57 31


https://github.com/user-attachments/assets/00654e00-37c7-4aed-bf06-611819f4cbaa

**Review by EmilStenstrom — APPROVED** [review]
Looks great, thanks for the video (including background music!)

**Pull request merged**